stripper/ze_aurora_tower_v6_6.cfg

```diff
@@ -1,3 +1,50 @@
+;fix people using button to jump over fence and early trigger
+add:
+{
+	"rendermode" "10"
+	"origin" "-5447.25 226.25 1037"
+	"angles" "0 0 0"
+	"targetname" "resize_me4"
+	"classname" "func_brush"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "resize_me4AddOutputsolid 20.5-1"
+		"OnMapSpawn" "resize_me4AddOutputmins -15.75 -13.25 -731-1"
+		"OnMapSpawn" "resize_me4AddOutputmaxs 13.25 15.75 731-1"
```

Comment:
Have you mixed up the x/y values between mins/maxs here?

Also keep in mind AddOutput mins/maxs for solid objects should only be used for things out of playable area players won't normally touch, as clients cannot predict them. Existing models need to be used if proper prediction is needed on new solids.

Reply:
My original intentions with the `func_brush` was to block off the top of the button as seen in this [image](https://cdn.discordapp.com/attachments/1097016521546543216/1130422484953485312/image.png). But now that you explained, I redid the entire system by blocking off the entire fence and then killing it as the elevator goes up. I delayed killing it by 1 second after the elevator leaves which should properly account for people not early triggering still but people on elevator wouldn't get crushed.